addons/web/static/src/core/notifications/notification.scss

```diff
@@ -7,6 +7,7 @@
     z-index: $o-notification-zindex;
 
     .o_notification {
+        background-color: var(--website-notification-background-color, #{$o-view-background-color}); // Needed to override the background in 'website.scss'
```

Comment:
I wouldn't use a variable name specific to website...

Reply:
Hello, thanks for the review. You are right, I had to refresh my mind in the guidelines...
Does it looks fine to you now ?